src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php

```diff
@@ -83,6 +83,10 @@ private static function getPhpUnitCliConfigArgument()
                 $argPath = substr($testArg, strlen('--configuration='));
                 $dir = realpath($argPath);
                 break;
+            } elseif (strpos($testArg, '-c') === 0) {
```

Comment:
However, the condition should be reversed: `0 === strpos($testArg, '-c')`. We use Yoda conditions in Symfony (the previous condition should be fixed as well).